5.2 Ledger State.md

```diff
@@ -1,13 +1,25 @@
 # 5.2 Ledger State
 
+## 5.2.1 Introduction
+
 The introduction of a voting-based consensus requires a fast and easy way to determine a node's initial opinion for every received transaction. This includes the ability to both detect double spends and transactions that try to spend non-existing funds. 
 These conditions are fulfilled by the introduction of an Unspent Transaction Output (UTXO) model for record-keeping, which enables the validation of transactions in real time, see also the section on [UTXO](./5.1%20UTXO.md).
 
 The concept of UTXO style transactions is directly linked to the creation of a directed acyclic graph (DAG), in which the vertices are transactions and the links between these are determined by the outputs and inputs of transactions. 
 
 To deal with double spends and leverage on certain properties of UTXO, we introduce the Realities Ledger State. 
 
-## 5.2.1 Realities Ledger State 
+
+## 5.2.2 Dependencies
+
+The *Ledger State* depends on:
+
+- UTXO: see the Section on [UTXO DAG](#5.2.4%20The%20UTXO%20DAG) as well as [5.1 UTXO](./5.1%20UTXO.md).
+- Tangle: the Tangle maps the approval relations between messages as well as transactions, see [4.1 The Tangle](./4.1%20The%20Tangle.md).
+- Solidification: Secures that all non-conflicting transactions converge to the same ledger state, see [4.4 Solidification](./4.4%20Solidification.md). 
+
```

Comment:
Should tip-selection be mentioned here (not sure)?

Reply:
I would think that the tip selection depends on the ledger state but not the other way around. Is there a particular point in this document that creates this dependency?